src/core.c/CompUnit/RepositoryRegistry.rakumod

```diff
@@ -175,10 +175,14 @@ class CompUnit::RepositoryRegistry {
             $home-spec = 'inst#' ~ $home;
         }
 
+        my $reloc = nqp::gethllsym('default', 'SysConfig').rakudo-build-config()<static-rakudo-home> eq '';
+        my $wrapper-mode = $reloc ?? 'relative' !! 'absolute';
```

Comment:
Should we use an enum for these?

Reply:
I don't know why, but I didn't manage to make enums work in this file. Bootstrap issues? Do you know if there is a way around that?